timezones/full/build.gradle.kts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019-2024 JetBrains s.r.o. and contributors.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+import TzdbTasks.ConvertType
+import TzdbTasks.InstallTimeTzdb
+import TzdbTasks.TzdbDownloadAndCompile
+import org.jetbrains.kotlin.gradle.targets.js.dsl.ExperimentalWasmDsl
+import org.jetbrains.kotlin.gradle.targets.js.npm.NpmResolverPlugin
+
+plugins {
+    kotlin("multiplatform")
+    id("maven-publish")
+}
+
+val tzdbVersion: String by properties
+val timezonesMajorVersion: String by properties
+version = "$timezonesMajorVersion.$tzdbVersion"
+
+val convertedKtFilesDir = File(project.buildDir, "wasmWasi-full/src/internal/tzData")
+val tzdbDirectory = File(project.projectDir, "tzdb")
+
+kotlin {
+    @OptIn(ExperimentalWasmDsl::class)
+    wasmWasi {
+        nodejs()
+        NpmResolverPlugin.apply(project) //Workaround KT-66373
+    }
+
+    sourceSets {
+        commonMain {
+            dependencies {
+                compileOnly(project(":kotlinx-datetime"))
```

Comment:
Applying timezones without datetime lead to broken compilation (or exception on usage times zone when PL is enabled). This helps us to avoid version collisions on gradle side and circullar dependency while testing.